iterator.go

```diff
@@ -44,7 +44,7 @@ func (it *ScanIterator) Next(ctx context.Context) bool {
 		}
 
 		// Fetch next page.
-		if it.cmd.args[0] == "scan" {
+		if it.cmd.args[0] == "scan" || len(it.cmd.args) < 3 {
```

Comment:
Could you instead specify the command here? Something like

```
switch it.cmd.args[0] {
    case "scan", "foo", "bar":
       it.cmd.args[1] = it.cmd.cursor
    default:
        it.cmd.args[2] = it.cmd.cursor
}
```

It seems to be easier to maintain it this way.